packages/api/src/external/commonwell/admin/recreate-patients-at-hies.ts

```diff
@@ -97,14 +97,19 @@ export async function recreatePatientAtCW(
 
     // create new patient, including linkint to person and network link to other patients
     log(`Creating new patient at CW...`);
-    const { commonwellPatientId: newCWPatientId, personId: newPersonId } = await create(
-      patient,
-      facilityId,
-      {
-        organization,
-        facility,
-      }
-    );
+    const cwIds = await create(patient, facilityId, {
+      organization,
+      facility,
+    });
+
+    if (!cwIds) {
+      const msg = `Missing CW IDs while recreating patient at CW`;
+      log(msg);
+      capture.message(msg, { extra: { patientId: patient.id }, level: "error" });
+      return undefined;
+    }
```

Comment:
I don't think we should send an alert/notification if a cx has the FF disabled. I suggest we add a check at the start of this function as well to avoid that.